Dockerfile.package

```diff
@@ -22,6 +23,9 @@ ARG KONG_PACKAGE_NAME
 ARG KONG_CONFLICTS
 ARG BUILDPLATFORM
 
+RUN mkdir -p /tmp/build/lib/systemd/system/
+COPY kong.service /tmp/build/lib/systemd/system/kong.service

```

Comment:
Not a blocker: would `/etc/kong` be a better location for this file, similarly to the template configuration file that we ship?

Reply:
(my understanding is) we're putting the default Kong `kong.service` file in `/lib/systemd/system` but then advise the users to customize Kong via `/etc/systemd/system/kong.service` file (ie https://docs.konghq.com/enterprise/1.3-x/systemd/#customize-the-kong-enterprise-unit-file )

FWIW I've very liberally stole most f what you see from https://github.com/Kong/kong-distributions/pull/263 and next sprint will need to PR doc changes that resemble. I have no strong feelings but keeping the two syncronized seems ideal

Reply:
It seems misleading to place a configuration file template in the `lib` directory - typically meant to contain shared and/or static libraries. This is, in fact, subject of a Linux standard - the "Filesystem Hierarchy Standard" . But, not a blocker.